include/GafferImageUI/ImageView.h

```diff
@@ -108,7 +114,9 @@ class ImageView : public GafferUI::View
 		static void registerDisplayTransform( const std::string &name, DisplayTransformCreator creator );
 		static void registeredDisplayTransforms( std::vector<std::string> &names );
 
-	protected :
+        ImageViewSignal &zoomLevelChangedSignal();
```

Comment:
I'm using a signal here so that I can block the connection temporarily when updating the plug if the user manually zoom in the viewport.